### DISCUSSION TITLE
🐛 Unknown option '--bitcode' error

### DISCUSSION THREAD

**Discussion by gf0x** [swiftyfinch/Rugby#105] [Q&A] [answered]
## Did you already try Rugby doctor?
yes

## Describe the bug
Unknown option '--bitcode' error

## To Reproduce
1. rugby cache -c Debug -c Release -s sim ios --bitcode 
2.⛔️ Error: Unknown option '--bitcode'

## Expected behavior
Cache succeeded

## Environment (please complete the following information):
 - Version 1.14.2

**Comment by swiftyfinch** [comment, marked as answer]
@gf0x Hello! Please, use the next version 1.15.0. I've just released it.

Also, you can find actual docs for version 1.14.2 here https://github.com/swiftyfinch/Rugby/tree/1.14.2.
By default, GitHub shows you docs for the main branch.

**Reply by gf0x** [reply]
Thanks a lot for a quick reply!
I did notice that `--bitcode` is a new feature so it seemed to me that it could not have been released yet. I wanted to point that it's really needed, however.

Thanks a lot for your great job!

**Reply by swiftyfinch** [reply]
You are welcome!
